routes/index.js

```diff
@@ -379,16 +401,20 @@ module.exports = function (passport) {
     if (!req.body) {
       return res.sendStatus(400);
     }
-    var badgeID = req.params.badge;
-
+    let badgeID = req.params.badge;
     dbAPI.validateUser(badgeID).then(function (ret) {
       if (ret.status == "User" && ret.status != "Admin") {
         dbAPI.modifyUser(badgeID, { status: "Manager" }).then(function (result) {
           if (result == undefined) {
             req.flash("error", "Internal error: unable to promote " + badgeID);
           }
           else {
-            req.flash("success", "Successfully promoted user associated with Badge number " + badgeID);
+            let adminName = req.user.first + ' ' + req.user.last;
+            let adminEmail = req.user.email;
+            let adminBadge = req.user.badge;
+            dbAPI.logEvent('administration', adminBadge, `Admin ${adminName} (${adminEmail}) has promoted user with badge ID: ${badgeID} to Manager status`);
+            dbAPI.logEvent('privilege', badgeID, `User with badge ID: ${badgeID} has been promoted to Manager status`);
+            req.flash("success", "Successfully promoted user associated with badge number " + badgeID);
```

Comment:
Can u add req.flash('fade_out','3000') here after this line because I forget to add.

Reply:
I can fix this

Reply:
@BishoyHanna91 should be fixed now